src/routes/+layout.svelte

```diff
@@ -105,10 +105,17 @@
 		}
 		try {
 			isLoading = true;
+
+			const resSettings = await getSettings();
+			if (resSettings.status === 'error') {
+				throw new Error('Could not get settings from database');
+			}
+			$settings = resSettings.settings;
+			$buoys = resSettings.buoys;
+
 			const oneWeekAgo = dayjs().locale('en-US').subtract(7, 'day').format('YYYY-MM-DD');
 			// TODO: this is super slow
-			const [resSettings, resAppData] = await Promise.all([
-				getSettings(),
+			const [resAppData] = await Promise.all([
```

Comment:
no need for promise.all on this one

Reply:
I added a new commit that allows for getSettings to still be called within promise.all.  Please check if you think it's a better solution.